### PR TITLE
Add expansion pack pets and food, beta compatibility

### PR DESCRIPTION
**Pull request by BenMiller3**
Tested on the beta version https://teamwood.itch.io/super-auto-pets-test and everything is working.

Would love a second look on some of the naming of the pets just to make sure that I'm consistent with what the game displays to avoid confusion.

### CODE REVIEW

File: start-proxy.ps1 [hunk 1]
```diff
@@ -1 +1 @@
-mitmproxy --allow-hosts api.teamwood.games -s mitmproxy-plugin.py
+mitmproxy --allow-hosts teamwood.games -s mitmproxy-plugin.py
```
Comment (charlie-collard) [inline start-proxy.ps1:1]: Has the domain also changed for the non-beta version?

**Review by charlie-collard — COMMENTED** [review]
Wow, I know this must have been a lot of work, thank you!

I’m away from my laptop at the moment but will do some testing soon.

File: start-proxy.ps1 (hunk 1 above)
Comment (BenMiller3) [reply start-proxy.ps1:1]: It's the same, but this domain prefix is able to capture both staging.teamwood.games and api.teamwood.games

**Review by charlie-collard — APPROVED** [review]
All seems to work, amazing. Thank you again!

**Pull request merged**